src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import { jwtService } from '../services/jwt.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    return res.sendStatus(401);
+  }
+
+  const userData = jwtService.verifyToken(token);
+
+  if (!userData) {
+    return res.sendStatus(401);
+  }
+
+  next();
```

Comment:
You should attach the `userData` to the `req` object before calling `next()`. This way, the subsequent middleware or route handlers can access the authenticated user's data.